### ISSUE TITLE
Option `--insecure-registry` does not work with login

### ISSUE THREAD

**Issue by yipengh** [containerd/nerdctl#284]
Hi,

I'm trying to use nerdctl with a private harbor without TLS, while the global flag `--insecure-registry` does not really ignore the HTTP responses.

Here is what i didi:
```
nerdctl --insecure-registry login -u admin 10.101.72.12:30002
```

and what i got:
![image](https://user-images.githubusercontent.com/4461484/124852699-4562d600-dfd7-11eb-9aed-6e4591f49e99.png)

what i got with `--debug` flag:
```
# nerdctl --insecure-registry --debug login -u admin 10.101.72.12:30002
Enter Password: DEBU[0003] hostDir: /etc/docker/certs.d/10.101.72.12:30002
DEBU[0003] attempting v2 login to registry endpoint https://10.101.72.12:30002/v2/
INFO[0003] Error logging in to endpoint, trying next endpoint  error="Get \"https://10.101.72.12:30002/v2/\": http: server gave HTTP response to HTTPS client"
FATA[0003] Get "https://10.101.72.12:30002/v2/": http: server gave HTTP response to HTTPS client
```

Thanks in advance.

**Comment by fahedouch** [comment]
Hello @yipengh , 

yep `--insecure-registry` is not yet implemented for login command. I will fix this :)

**Comment by AkihiroSuda** [comment]
Fixed in #286 

thanks @fahedouch
